RecoEgamma/PhotonIdentification/interface/PhotonIsolationCalculator.h

```diff
@@ -7,11 +7,23 @@
 #include "DataFormats/EgammaCandidates/interface/Photon.h"
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "Geometry/CaloTopology/interface/HcalTopology.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/HcalSeverityLevelComputer.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/HcalSeverityLevelComputerRcd.h"
+#include "CondFormats/HcalObjects/interface/HcalChannelQuality.h"
+#include "CondFormats/DataRecord/interface/HcalChannelQualityRcd.h"
```

Comment:
can you double check that these are in the plugins buildfile? by a quick look, several are missing.

Reply:
Ok odd, was thinking if it runs then it's not needed. Will fix.

Reply:
These days we require indeed that all the dependencies are explicitly stated, as it simplifies maintenance later.